java/src/jmri/jmrit/logixng/actions/swing/DigitalActionSwingBundle.properties

```diff
@@ -173,6 +183,10 @@ DigitalFormula_InvalidFormula           = Formula is invalid: "{0}"
 
 EnableLogix_Components              = Set Logix {0} to {1}
 
+EnableLogixNG_Components            = Set LogixNG {0} to {1}
+EnableLogixNG_Info1 = Enable and disable is used if you want the change to be stored in the panel file.
```

Comment:
A layout data file (aka panel file ) has zero or more panels.  No change is needed since "panel file" appears to be a permanent part of JMRI terminology.

Reply:
Thanks. I try to use that terminology but missed it in this case. I have fixed it now.